lib/middleware/request_tracker.rb

```diff
@@ -213,7 +213,7 @@ def call(env)
     if error_details = rate_limit(request, cookie)
       available_in, error_code = error_details
       message = <<~TEXT
-        Slow down, too many requests from this IP address.
+        Slow down, too many requests from this user or IP address.
```

Comment:
How about we just say which it is?

Reply:
I didn't because it required slightly more refactoring. But you're right, this will be useful. Done :+1:

Reply:
Brought to by the "never make a person think if the computer can easily do the work" department